Android/app/src/main/java/io/github/project_travel_mate/utilities/ActivityConversionListview.java

```diff
@@ -0,0 +1,95 @@
+package io.github.project_travel_mate.utilities;
+
+import android.app.Activity;
+import android.app.ProgressDialog;
+import android.content.Context;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.widget.ListView;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.StringTokenizer;
+
+import butterknife.BindView;
+import io.github.project_travel_mate.R;
+import objects.CurrencyName;
+
+public class ActivityConversionListview extends Activity {
+
+    @BindView(R.id.listView)
+    ListView listview;
+    CurrencyConverterAdapter adaptr_listview;
```

Comment:
We have used camelcase for variable names everywhere. You also use those only

Reply:
Done